db/db_properties_test.cc

```diff
@@ -1713,6 +1716,44 @@ TEST_F(DBPropertiesTest, BlockCacheProperties) {
   ASSERT_EQ(0, value);
 }
 
+TEST_F(DBPropertiesTest, TableCacheProperties) {
+  Options options;
+  uint64_t value, new_value;
+
+  options.env = CurrentOptions().env;
+
+  // Block cache properties are not available if block cache is not used.
+  BlockBasedTableOptions table_options;
```

Comment:
There is already a default options, so you should not need to create the table_options here...

Reply:
Previous tests make changes.  Thought this was a way to guarantee the traditional .sst tables.  Because I know how to force creation of a new .sst file with the traditional.  Not sure if same rules apply with others.

Reply:
The redundant table factory creation is now gone.